slo-monitor/README.md

```diff
@@ -0,0 +1,15 @@
+# Kubernetes Performance SLO monitor
+
+This tool monitors Performance SLOs that are unavailable from the API server and exposes Prometheus metrics for them.
+
+It can be run anywhere and only requirement is that it needs to be able to talk with the API server.
+
+# Usage
+
+SLO monitor is a simple pod that needs to be able to read Pods and Events. On top of that it should work as long as it can communicate with the API server.
+
+When starting the monitor you need to set memory limit, as by design it has to be restarted from time to time. It is so because we had two choices:
```

Comment:
The fact that it needs to be restarted doesn't sound like a good design to me.
If i really needs, maybe it just can exit itself from time to time?